src/gui/hotkeyConfig.cpp

```diff
@@ -288,17 +288,17 @@ std::vector<std::pair<string, string>> HotkeyConfig::listHotkeysByCategory(strin
                 for(auto key_name : sfml_key_names)
                 {
                     if (key_name.second == item.hotkey.code){
-			string keyModifier = "";
-			if (item.hotkey.shift) {
-				keyModifier = "Shift+";
-			} else if (item.hotkey.control) {
-				keyModifier = "Ctrl+";
-			} else if (item.hotkey.alt){
-				keyModifier = "Alt+";
-			}
+            string keyModifier = "";
```

Comment:
This looks wrongly indented now

Reply:
Actually, it was wrongly indented before, I only converted the tabs to spaces.

But the additional commit now also fixes that.